ddp-workspace/projects/ddp-dsm-ui/src/app/scan/scan.component.ts

```diff
@@ -110,20 +108,22 @@ export class ScanComponent implements OnInit {
 
   private validateValue(labelValue: string, position: number, isLeft: boolean): boolean {
     let isDuplicate = false;
```

Comment:
As far as I understand, this function checks whether values are duplicated or not, hence this function name (validateValue) is misleading. I would suggest renaming it to 'isDuplicate' or something similar.

Reply:
Done diddly did 👍